pkgs/development/python-modules/functiontrace/default.nix

```
@@ -0,0 +1,27 @@
+{ lib
+, buildPythonPackage
+, fetchPypi
+, toml
+}:
+
+buildPythonPackage rec {
+  pname = "functiontrace";
+  version = "0.3.7";
+  format = "setuptools";
```

This actually has a pyproject.toml, even if it still uses the setuptools build system. So it should probably be `pyproject` I think?

```suggestion
  format = "pyproject";
```